src/controllers/userProfile.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-cond-assign */
+/* eslint-disable no-else-return */
+/* eslint-disable prefer-destructuring */
+import models from '../models';
+
+const { User } = models;
+/**
+  * This class contains user controllers
+  */
+class userProfile {
+  /**
+    * @param  {object} req
+    * @param  {object} res
+    * @return {object} returns an object containing the user profile
+    */
+  static async getProfile(req, res) {
+    const { username } = req.params;
+    const queryResult = await User.findOne({
+      where: { username }
+    });
+    if (!queryResult) {
+      return res.status(404).json({ message: `Username: ${username} does not exist` });
+    }
+    const profile = queryResult.dataValues;
```

Comment:
I don't see the need for this variable since we have that same result stored in the queryResult variable